.github/workflows/integration-tests.yml

```diff
@@ -287,15 +291,20 @@ jobs:
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
         with:
           ref: ${{ github.event.pull_request.head.sha }}
-      ## Run this step when changes that require tests to be run are made
+      - name: Get Latest Version
+        id: get_latest_version
+        run: |
+          echo "latest_version=$(curl --header "Authorization: token ${{ secrets.GITHUB_TOKEN }}" --request GET https://api.github.com/repos/${{ github.repository }}/releases/latest | jq -r .name)"  >> "$GITHUB_OUTPUT"
```

Comment:
You'll want to strip out the `v` from the version tag (v1.2.3) either here or later as the Docker image doesn't have that prefix (1.2.3)